### ISSUE TITLE
Add support for Minds

### ISSUE THREAD

**Issue by wilddeej** [aurelg/feedspora#59]
The Minds social media platform would likely be a good addition for FeedSpora.  A Python API for Minds can be found at https://gitlab.com/granitosaurus/minds-api, and there may be others as well.

**Comment by aurelg** [comment]
Absolutely. Thanks for pointing out this network. I did not know it. :)

**Comment by wilddeej** [comment]
I started work on this, and then happened upon [a distressing thread on Minds](https://www.minds.com/newsfeed/835509219409039360).  It seems that services like FeedSpora are not welcome on Minds, at least not at this point...

Perhaps they will change their... Minds... at some point in the future, but for now it seems best to leave this one unimplemented.
